README.md

```diff
@@ -1,15 +1,15 @@
 <div align="center">
 <img src="./bonaparte.webp" height="150">
 
-### ⚡ Cairo-rs ⚡
+### ⚡ Cairo-vm ⚡
 
 A faster and safer implementation of the Cairo VM in Rust
 
-[Report Bug](https://github.com/lambdaclass/cairo-rs/issues/new?labels=bug&title=bug%3A+) · [Request Feature](https://github.com/lambdaclass/cairo-rs/issues/new?labels=enhancement&title=feat%3A+)
+[Report Bug](https://github.com/lambdaclass/cairo-vm/issues/new?labels=bug&title=bug%3A+) · [Request Feature](https://github.com/lambdaclass/cairo-vm/issues/new?labels=enhancement&title=feat%3A+)
```

Comment:
We should change this link to use the bug issue template:
```
https://github.com/lambdaclass/cairo-vm/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=bug%3A+
```

Reply:
good!
done